src/color-picker/README.md

```diff
@@ -37,6 +37,30 @@ You can use your component instead of the default color swatch:
 </color-picker>
 ```
 
+or the default space picker:
+
+```html
+<color-picker space="oklch" color="oklch(60% 30% 180)">
+	<select slot="space-picker" size="3" class="horizontal">
+		<option value="oklch" selected>OKLCh</option>
+		<option value="hwb">HWB</option>
+		<option value="hpluv">HPLuv</option>
+	</select>
+</color-picker>
+
+<style>
+	select.horizontal {
+		writing-mode: tb;
+		field-sizing: content;
```

Comment:
The horizontal select trick is nice, but not a good option for a demo, you want anything that is not relevant to the demo itself to be relatable, not to have people scratching their heads

Reply:
Got it. I'll rework it. Thanks!